File: console/android/ORLib/src/main/java/io/openremote/orlib/models/ORAppConfig.kt
package io.openremote.orlib.models

data class ORAppConfig(
    val id: Int,
    val realm: String,
    val initialUrl: String,
    val url: String,
    val menuEnabled: Boolean = false,
    val menuPosition: String = "BOTTOM_LEFT",
    val menuImage: String?,
    val primaryColor: String?,
    val secondaryColor: String?,
    val links: List<ORLinkConfig>?
)

File: console/iOS/ORLib/ORLib/Models/ORAppConfig.swift
//
//  ORAppConfig.swift
//  GenericApp
//
//  Created by Michael Rademaker on 21/10/2020.
//  Copyright © 2020 OpenRemote. All rights reserved.
//

import UIKit

public struct ORAppConfig: Codable {
    let realm: String
    let initialUrl: String?
    let url: String
    let menuEnabled: Bool
    let menuPosition: String?
    let primaryColor: String?
    let secondaryColor: String?
    let links: [ORLinkConfig]?
}
